packages/react/src/TreeView/TreeView.module.css

```diff
@@ -0,0 +1,255 @@
+    across more lines. */
+    /* stylelint-disable-next-line primer/spacing */
+    padding-top: calc(var(--min-item-height) / 2 - var(--base-size-12) / 2);
+    color: var(--fgColor-muted);
+    grid-area: toggle;
+    justify-content: center;
+    align-items: flex-start;
+  }
+
+  .PRIVATE_TreeView-item-toggle--hover:hover {
+    background-color: var(--color-tree-view-item-chevron-hover-bg);
+  }
+
+  .PRIVATE_TreeView-item-toggle--end {
+    border-top-left-radius: var(--borderRadius-medium);
+    border-bottom-left-radius: var(--borderRadius-medium);
+  }
+
+  .PRIVATE_TreeView-item-content {
+    display: flex;
+    height: 100%;
+    padding: 0 var(--base-size-8);
+
+    /* The dynamic top and bottom padding to maintain the minimum item height for single line items */
+    /* stylelint-disable-next-line primer/spacing */
+    padding-top: calc((var(--min-item-height) - var(--custom-line-height, 1.3rem)) / 2);
+    /* stylelint-disable-next-line primer/spacing */
+    padding-bottom: calc((var(--min-item-height) - var(--custom-line-height, 1.3rem)) / 2);
+    line-height: var(--custom-line-height, var(--text-body-lineHeight-medium, 1.4285));
+    grid-area: content;
+    gap: var(--stack-gap-condensed);
+  }
+
+  .PRIVATE_TreeView-item-content-text {
+    flex: 1 1 auto;
+    width: 0;
+  }
+
+  &[data-truncate-text='true'] .PRIVATE_TreeView-item-content-text {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+
+  &[data-truncate-text='false'] .PRIVATE_TreeView-item-content-text {
+    word-break: break-word;
+  }
+
+  .PRIVATE_TreeView-item-visual {
+    display: flex;
+
+    /* The visual icons should appear vertically centered for single-line items, but remain at the top for items that wrap
+    across more lines. */
+    height: var(--custom-line-height, 1.3rem);
+    color: var(--fgColor-muted);
+    align-items: center;
+  }
+
+  .PRIVATE_TreeView-item-leading-action {
+    display: flex;
+    color: var(--fgColor-muted);
+    grid-area: leadingAction;
+  }
+
+  .PRIVATE_TreeView-item-level-line {
+    width: 100%;
+    height: 100%;
+
+    /*
+     * On devices without hover, the nesting indicator lines
+     * appear at all times.
+     */
+    border-color: var(--borderColor-muted);
+    border-right: var(--borderWidth-thin) solid;
+  }
+
+  /*
+   * On devices with :hover support, the nesting indicator lines
+   * fade in when the user mouses over the entire component,
+   * or when there's focus inside the component. This makes
+   * sure the component remains simple when not in use.
+   */
+  @media (hover: hover) {
+    .PRIVATE_TreeView-item-level-line {
+      border-color: transparent;
+    }
+
+    &:hover .PRIVATE_TreeView-item-level-line,
+    &:focus-within .PRIVATE_TreeView-item-level-line {
+      border-color: var(--borderColor-muted);
+    }
+  }
+
+  .PRIVATE_TreeView-directory-icon {
+    display: grid;
+    color: var(--treeViewItem-leadingVisual-iconColor-rest);
```

Comment:
Mapping: [`${get('colors.treeViewItem.directory.fill')}`](https://github.com/primer/react/pull/5267/files#diff-8bb4d65de0c1d20b6dfd22fba3aa211836b3af0aa504256ce69c55e7c04f085dL269) -->  [`var(--treeViewItem-leadingVisual-iconColor-rest)`](https://github.com/primer/primitives/blob/9f1033462619f7a3e3a191dcfc0a50cf7708c79a/src/tokens/fallback/color-fallbacks.json#L183)

Reply:
See https://github.com/primer/react/pull/5267#issuecomment-2474389109 for vrt failures related to this variable: https://github.com/primer/react/pull/5267#issuecomment-2474389109